src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -759,19 +762,21 @@ Namespace Microsoft.CodeAnalysis.Operations
         End Function
 
         Private Function CreateBoundWithLValueExpressionPlaceholder(boundWithLValueExpressionPlaceholder As BoundWithLValueExpressionPlaceholder) As IInstanceReferenceOperation
+            Dim referenceKind As InstanceReferenceKind = InstanceReferenceKind.InitializedInstance
```

Comment:
>InitializedInstance [](start = 79, length = 19)

I think this could be an implicit receiver in an With statement. We probably should have a special kind for this case and use the appropriate one here.

Reply:
Addressing this by renaming `InitializedInstance` to `ImplicitReceiver`. If we later get a request to be able to tell whether the instance is actually being initialized or just used as part of a With statement, then we can revisit or add some other boolean property.

---
In reply to: [180531576](https://github.com/dotnet/roslyn/pull/26055#discussion_r180531576) [](ancestors = 180531576)